modules/programs/bashmount.nix

```
@@ -0,0 +1,34 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.bashmount;
+
+in {
+  meta.maintainers = [ maintainers.AndersonTorres ];
+
+  options.programs.bashmount = {
+    enable = mkEnableOption "bashmount";
+
+    settings = mkOption {
```

The `settings` name is typically reserved for structured configuration à la [RFC 42](https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md#part-1-structural-settings-instead-of-stringly-extraconfig). For stringly configuration we keep using `extraConfig`:

```suggestion
    extraConfig = mkOption {
```

---

I have used `rtorrent.nix` as example. Is there any difference?

---

No different, I think the option in the rtorrent module should have been `extraConfig` as well. Just some mistake.